fr-FR/lang/numere.nlng

```diff
@@ -0,0 +1,1186 @@
+# GNU General Public License pour plus de détails.
+#
+# Vous devriez avoir reçu une copie de la Licence Publique Générale GNU
+# avec ce programme.  Si ce n'est pas le cas, voir <http://www.gnu.org/licenses/>.
+# ========================================================================
+# LANGUAGE FILE: numere.nlng
+LANGUAGE: FR
+#
+# PARSERFUNCS:
+#======================================
+PARSERFUNCS_LISTFUNC_HEADLINE=Vordefinierte Funktionen
+PARSERFUNCS_LISTFUNC_TYPE_NUM=Numerik
+PARSERFUNCS_LISTFUNC_TYPE_MAT=Matrix und Vektor
+PARSERFUNCS_LISTFUNC_TYPE_TRIGONOMETRIC=Trigonometrie
+PARSERFUNCS_LISTFUNC_TYPE_HYPERBOLIC=Hyperbelfunktionen
+PARSERFUNCS_LISTFUNC_TYPE_POLYNOMIAL=Polynome
+PARSERFUNCS_LISTFUNC_TYPE_STATS=Statistik
+PARSERFUNCS_LISTFUNC_TYPE_STRING=Zeichenketten
+PARSERFUNCS_LISTFUNC_TYPE_LOGIC=Logik
+PARSERFUNCS_LISTFUNC_TYPE_TIME=Zeit
+PARSERFUNCS_LISTFUNC_TYPE_RANDOM=Zufall
+PARSERFUNCS_LISTFUNC_TYPE_PHYSICS=Physik
+PARSERFUNCS_LISTFUNC_TYPE_ANGULAR=Winkel
+PARSERFUNCS_LISTFUNC_TYPE_DISTRIB=Distributionen
+PARSERFUNCS_LISTFUNC_TYPE_LOGARITHMIC=Logarithmen
+PARSERFUNCS_LISTFUNC_TYPE_COORDS=Koordinaten
+PARSERFUNCS_LISTFUNC_TYPE_DRAW=Zeichnungen
+PARSERFUNCS_LISTFUNC_TYPE_DEFINE=Eigene Definitionen
+PARSERFUNCS_LISTFUNC_TYPE_GUI=Graphische Oberflächen
+PARSERFUNCS_LISTFUNC_TYPE_CMPLX=Komplexe Zahlen
+# Funktionentabelle
+DEFINED_FOR_ALL=Definiert für alle reellen oder komplexen Zahlen
+DEFINED_FOR_ALL_C_RANGE_R=Definiert für alle komplexen Zahlen oder für reelle Zahlen im Bereich
+DEFINED_REAL=Definiert für alle reellen Zahlen
+DEFINED_NATURAL=Definiert für natürliche Zahlen
+PARSERFUNCS_LISTFUNC_FUNC_ABS_[NUM]=						abs(x)                                                      VAL      - Montant du nombre réel ou complexe x
+PARSERFUNCS_LISTFUNC_FUNC_ACOSH_[NUM_HYPERBOLIC]=			acosh(x)                                                    VAL      - Areakosinus Hyperbolicus de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [1,inf) (synonyme : "arcosh(x)")
+PARSERFUNCS_LISTFUNC_FUNC_ACOS_[NUM_TRIGONOMETRIC]=			acos(x)                                                     VAL      - Arcuscosinus de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [-1,1] (synonyme : "arccos(x)")
+PARSERFUNCS_LISTFUNC_FUNC_ACSCH_[NUM_HYPERBOLIC]=			acsch(x)                                                    VAL      - Hyperbolicus d'Areakosekan de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [1,inf]
+PARSERFUNCS_LISTFUNC_FUNC_ACSC_[NUM_TRIGONOMETRIC]=			acsc(x)                                                     VAL      - Arkuskosekans de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [-1,1]
+PARSERFUNCS_LISTFUNC_FUNC_AI_[NUM_PHYSICS]=					Ai(x)                                                       VAL      - Fonction Airy Ai(x). %%DÉFINI_RÉEL%%
+PARSERFUNCS_LISTFUNC_FUNC_ALPHA_STABLE_RD_[RANDOM_DISTRIB]= alpha_stable_rd(c,alpha)                                    VAL      - Liefert eine Zufallszahl aus der stabilen Levy-Alpha-Verteilung mit Skalenparameter c und Exponent alpha.
```

Comment:
Not translated?

Reply:
Here again it can not be detected, because the "table structure" is not available. (not enough spaces between = and alpha....)

Reply:
The principle is `\talpha_levy...()    VAL    - TEXT`, so the first space is a tab character. But as I mention in my other reply, I'd use the ` - ` marker for finding the start of the documentation (which is also valid in this case). For reference, in `kernel/core/ui/calltipprovider.cpp` all code regarding the extraction and processing of editor calltips is gathered. Read and reuse that to solve the problem